styles.css

```diff
@@ -0,0 +1,520 @@
+	font-weight: 500;
+}
+
+.accent-serif {
+	font-family: 'Gelasio', serif;
+}
+
+nav {
+	background-color: #C0D7E0;
+	padding: 10px 0;
+	overflow: auto;
+}
+
+a {
+	display: inline-block;
+	text-decoration: none;
+	color: #0E1116;
+}
+
+.nav-links {
+	overflow: auto;
+	width: 800px;
+	margin: 0 auto;
+	padding: 0 30px;
+	font-size: 20px;
+	font-family: 'Poppins', sans-serif;
+	text-transform: uppercase;
+}
+
+nav ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
+	line-height: 23px;
+}
+
+nav li {
+	display: block;
+	float: left;
+	width: 33.2%;
+	text-align: center;
+	margin: 2px 0 0 0;
+}
+
+section.showcase, section.about, section.portfolio {
+	height: calc(100vh - 235px);
+	position: relative;
+}
+
+section.showcase {
+	background-image: url('./images/project-1-hero.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+	background-size: cover;
+	text-align: center;
+	margin-top: 160px;
+}
+
+.hero-caption {
+	display: inline-block;
+	text-align: center;
+	position: absolute;
+	width: 575px;
+	top: 45%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	color: #E6FAFC;
+	background-color: rgba(14, 17, 22, 0.7);
+	padding: 20px 50px;
+}
+
+.hero-caption h1 {
+	font-size: 95px;
+}
+
+.a-box {
+	display: inline-block;
+	text-align: center;
+	color: #E6FAFC;
+	background-color: rgba(14, 17, 22, 0.7);
+	padding: 15px 30px;
+}
+
+.a-section-title {
+	margin-left: 20px;
+}
+
+.a-section-anchor {
+	position: absolute;
+	margin-top: -160px;
+	outline: none;
+}
+
+section.showcase .a-box {
+	font-size: 20px;
+	position: absolute;
+	bottom: 10%;
+	left: 70%;
+	transform: translate(-50%, 0);
+}
+
+section.about {
+	background-color: #374B68;
+	min-height: 600px;
+}
+
+.center-container {
+	width: 880px;
+	margin: 0 auto;
+	height: 100%;
+}
+
+.about-content {
+	overflow: auto;
+	min-height: 470px;
+}
+
+.about-content p {
+	width: 50%;
+	float: left;
+	padding: 0 30px;
+}
+
+.headshot {
+	display: block;
+	width: 46%;
+	float: left;
+	border: 20px solid #20385E;
+	border-radius: 50%;
+	margin: 20px 2%;
+}
+
+p {
+	font-size: 20px;
+	word-spacing: 2px;
+}
+
+section.portfolio {
+	background-color: #20385E;
+	min-height: 570px;
+}
+
+.touchscreen-info {
+	display: none;
+}
+
+.portfolio-content {
+	overflow: auto;
+	height: 83%;
+}
+
+.portfolio-project {
+	width: 33.3%;
+	height: 50%;
+	min-height: 193px;
+	float: left;
+	position: relative;
+	overflow: hidden;
+    background-repeat: no-repeat;
+	background-size: cover;
+}
+
+.project1 {
+	background-image: url('./images/project-1-sml.jpg');
+}
+
+.project2 {
+	background-image: url('./images/project-2-sml.jpg');
+}
+
+.project3 {
+	background-image: url('./images/project-3-sml.jpg');
+}
+
+.project4 {
+	background-image: url('./images/project-4-sml.jpg');
+}
+
+.project5 {
+	background-image: url('./images/project-5-sml.jpg');
+}
+
+.project6 {
+	background-image: url('./images/project-6-sml.jpg');
+}
+
+.project-caption {
+	width: 100%;
```

Comment:
only need 1 out of the width and height so remove the other

Reply:
Removing height makes the dark background disappear, but width can be removed.